baseplate/observers/prometheus.py

```diff
@@ -60,7 +61,7 @@ def set_metrics_by_protocol(self) -> None:
         if self.protocol == "thrift":
             self.metrics = PrometheusThriftServerMetrics()
         elif self.protocol == "http":
-            logger.warning("HTTP Prometheus metrics not implemented")
+            self.metrics = PrometheusHTTPServerMetrics()
```

Comment:
Possible to add a couple of tests? Otherwise looking good to me 👍

Reply:
Actually, just putting that in this PR since it's complete.